src/demo/native/story.js

```diff
@@ -59,21 +121,52 @@ storiesOf('Demos', module)
           width: 40px;
           height: 40px;
         }
+        .input-field {
+          height: 40px;
+          padding: 0 10px 0 20px;
+          min-width: 180px;
+          border: 1px solid;
+          font-size: 14px;
+          outline: none;
+          border-color: #e5e5e5; /* $color-prim-gray-mercury */
+          background-color: #fafafa; /* $color-prim-gray-alabaster */
+          color: #999; /* $color-prim-gray-dusty */
+          fill: #999; /* $color-prim-gray-dusty */
+        }
+        .input-field:hover {
+          border: 1px solid #00008f;
+        }
+        .input-field:focus {
+          outline: 2px solid #00008f;
+          outline-offset: -2px;
+          color: #000;
+        }
       </style>
       <h2>
-        Note: There are green border around each SVG to better see dimensions.
+        Note: The green borders reveal the dimensions of the SVGs.
       </h2>
-      <h3>${icons.length} Icons:</h3>
-      ${svg(
-        icons.map(
-          i =>
-            `<div>${i.svgstring}<span style="padding-left: 10px;">${
-              i.path
-            }</span></div>`
-        )
-      )}
+      <div>
+        <input
+          class="input-field"
+          type="text"
+          placeholder="Find icon/image"
+          oninput="onCallbackInput(arguments[0])"
+        />
+      </div>
+
+      <h3 class="iconHeader">${icons.length} Icons:</h3>
+      <div class="icons">
+        ${svg(
+          icons.map(
+            i =>
+              `<div>${i.svgstring}<span style="padding-left: 10px;">${
+                i.path
+              }</span></div>`
+          )
+        )}
+      </div>
 
-      <h3>${images.length} Images:</h3>
+      <h3 class="imageHeader">${images.length} Images:</h3>
```

Comment:
no camel case for classnames

https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/